NCIP/ncip-18.md

```diff
@@ -0,0 +1,50 @@
+---
+NCIP: 18
+Title: Retrieve FunngibleAssetValue transfer from avatar to agent
+Status: Draft
+Type: Core
+Author: ChunUng Yang <yang@planetariumhq.com>, Nine Chronicles team <9c-dev@planetariumhq.com> et al.
+Created: 2024-03-04
+---
+# Abstract
+
+This proposal aims to transfer [FungibleAssetValue] from belong avatar address to agent address.
+
+# Motivation
+
+If a [FungibleAssetValue](i.e, NCG) is sent to an incorrect avatar address instead of agent address(signer), the [FungibleAssetValue] is bound and there is no way to retrieve it. The agent who owns the avatar signs the Tx to take the [FungibleAssetValue] out of the avatar and make it moveable.
+
+# Specification
+
+To simplify, Move the NCG from the avatar address to the agent address through the new action `RetrieveAvatarAssets`.
+
+## `RetrieveAvatarAssets`
+
+The plain value is stored in Dictionary format like other actions. The schema is as follows.
+```
+{
+  "type_id": "retrieve_avatar_assets",              # action type name.
+  "values": [
+    AvatarAddress                                   # target avatar address from signer will retrieve assets.
+  ]
+}
+```
```

Comment:
Don't we have to specify what kind of FAV to retrieve?
Or this action retrieves all available FAVs from avatar to signer at once?

Reply:
For convenience, only NCG will be applied at first, and additional support will be processed within the action if necessary.